src/StateManager.ts

```diff
@@ -449,68 +481,70 @@ export default class StateManager {
   private static addStateAtDoubleClickPos(
     evt: Konva.KonvaEventObject<MouseEvent>,
   ) {
-    if (!StateManager._stage) return;
-
-    const stage = StateManager._stage;
-    const pointerPosition = stage.getPointerPosition();
-    if (!pointerPosition) return;
+    if (!this.debugActive) {
```

Comment:
Wrapping the whole function body in an if statement causes the diff to be quite large. I think it'd be better to instead provide an early-exit clause:
```ts
if (this.debugActive) return;
```
This way, the rest of the function can remain unchanged.